testutils/base.go

```diff
@@ -24,6 +24,13 @@ import (
 	"github.com/cockroachdb/cockroach/security"
 )
 
+const (
+	// TestUser is a fixed user used in unittests.
+	// It has a permissions config with read/write permissions
+	// on the "test-user" prefix.
+	TestUser = "test-user"
```

Comment:
Shouldn't this constant be defined next to `TestServer`. Or perhaps even inside of `TestServer` (i.e. `TestServer.User`).